css-layout-classes/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <title>CSS Grid Classes</title>
+  </head>
+  <body>
+   <div class="container">
+     <div class="row">
+       <div class="column-full">
+         <h1>Choose your character!</h1>
+       </div>
+     </div>
+     <div class="row">
+       <div class="column-one-third">
+         <img src="images/link.png" alt="Link">
+       </div>
+       <div class="column-one-third">
+         <h2>Super Smash Bros. Ultimate</h2>
+         <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics
+             calling it the best installment in the series. It received praise for its large amount of content and fine-tuning
+             of existing Smash gameplay elements, although its online mode was widely criticized.</p>
+       </div>
+       <div class="column-one-third">
+         <img src="images/yoshi.png" alt="Yoshi">
+       </div>
+    <div class="row">
+       <div class="column-one-fourth">
+         <img src="images/kirby.png" alt="Kirby">
+       </div>
+       <div class="column-one-fourth">
+         <img src="images/fox.png" alt="Fox">
+       </div>
+       <div class="column-one-fourth">
+         <img src="images/pikachu.png" alt="Pikachu">
+       </div>
+       <div class="column-one-fourth">
+         <img src="images/ness.png" alt="Ness">
+       </div>
+     </div>
+    </div>
+    <div class="row">
+     <div class="column-one-half">
+       <h2>Donkey Kong</h2>
+       <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum eligendi aperiam doloremque at veritatis, nemo tempora
+          facilis quas minus eius.</p>
+       <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Est, iure.</p>
+     </div>
+     <div class="column-one-half">
+       <img src="images/dk.png" alt="Donkey Kong">
+     </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half">
+        <img src="images/mario.png" alt="Mario">
+      </div>
+      <div class="column-one-half">
+        <h2>Mario</h2>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Esse similique nobis perspiciatis nihil aut quaerat eaque tempore
+           odit cumque repellendus.</p>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Et, impedit.</p>
+      </div>
```

Comment:
These `column-one-half` classes do not match the actual size these elements are meant to take up. You should make some more appropriate class names like `column-one-third` and `column-two-thirds`